tests/pages/donate.py

```diff
@@ -0,0 +1,10 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.page import Page
+
+
+class DonatePage(Page):
+
+    _url = 'https://donate.mozilla.org/'
```

Comment:
We could have the URL in the tests if that's preferred? I don't have a preference, and your concern is valid.